README.md

```diff
@@ -35,6 +35,91 @@ This flow chart illustrates the conversion process provided by `convert.py`. It
 
 The initial script is in place for the Verilog conversion steps.
 
+## Installation and Setup
+
+### Prerequisites
+
+- Python 3.10 or higher
+- Git
+- Build tools (gcc, make, etc.)
+- OpenAI API key (for LLM integration)
+
+### System Dependencies
+
+The project requires several system-level tools for formal verification and synthesis:
+
+- Yosys (for synthesis)
+- SymbiYosys (for formal verification)
+- EQY (for equivalence checking)
+- Z3 (SMT solver)
+
+### Quick Start
+
+1. Clone the repository:
+   ```bash
+   git clone https://github.com/stevehoover/conversion-to-TLV.git
+   cd conversion-to-TLV
+   ```
+
+2. Run the initialization script to install all dependencies:
+   ```bash
+   chmod +x init
+   ./init
+   ```
+
+3. Set up your OpenAI API key:
+   ```bash
+   export OPENAI_API_KEY='your-api-key-here'
+   ```
+
+### Manual Installation
```

Comment:
It's actually better not to have this section (most of it). A user can reference the script for the installation commands. Having them here runs the risk that they become obsolete.